GVFS/GVFS.Common/LocalGVFSConfig.cs

```diff
@@ -15,7 +15,7 @@ public class LocalGVFSConfig
 
         public LocalGVFSConfig()
         {
-            string servicePath = Paths.GetServiceDataRoot(GVFSConstants.Service.ServiceName);
+            string servicePath = GVFSPlatform.Instance.GetDataRootForGVFSComponent(GVFSConstants.Service.ServiceName);

```

Comment:
When we look at testing this, we should look at how to test these with these static instance variables (or if there is another pattern)